config/software/chef_backup-gem.rb

```diff
@@ -0,0 +1,26 @@
+#
+# Copyright:: Copyright (c) 2014 Chef Software, Inc.
+# License:: Apache License, Version 2.0
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+name 'chef_backup-gem'
+default_version '0.0.1.dev.2'
+
+dependency 'ruby'
+dependency 'rubygems'
+dependency 'rsync'
+
+build do
+  gem "install chef_backup -n #{install_dir}/embedded/bin --no-rdoc --no-ri -v #{version}"
```

Comment:
It's being pulled in from Rubygems.  I suppose we could pull it directly from git if that's preferred.